docs/admin/performance_tuning.md

```diff
@@ -79,62 +79,62 @@ $ self_test --help
 ```
 
 The RPC rate with empty request and reply is also often useful to evaluate what
 is the maximum capabilities of the network. This can be achieved as with the
 following command line:
 
 ```bash
-self_test -u --group-name daos_server --endpoint 0:2 --message-size "(0 0)" --max-inflight-rpcs 16 --repetitions 100000
+self_test --use-daos-agent-env --group-name daos_server --endpoint 0:2 --message-size "(0 0)" --max-inflight-rpcs 16 --repetitions 100000
 ```
 
 0 could be replaced with i2048 for instance to send a payload of 2Kb.
 
 All those 3 tests could be combined in a single and unique run:
 
 ```bash
-self_test -u --group-name daos_server --endpoint 0:2 --message-size "(0 0) (b1048578 0) (0 b1048578)" --max-inflight-rpcs 16 --repetitions 100000
+self_test --use-daos-agent-env --group-name daos_server --endpoint 0:2 --message-size "(0 0) (b1048578 0) (0 b1048578)" --max-inflight-rpcs 16 --repetitions 100000
 ```
 
 RPCs could also be send to a range of engine ranks and tags as follows:
 
 ```bash
-self_test -u --group-name daos_server --endpoint 0-<MAX_RANK>:0-<MAX_TAG> --message-size "(0 0) (b1048578 0) (0 b1048578)" --max-inflight-rpcs 16 --repetitions 100000
+self_test --use-daos-agent-env --group-name daos_server --endpoint 0-<MAX_RANK>:0-<MAX_TAG> --message-size "(0 0) (b1048578 0) (0 b1048578)" --max-inflight-rpcs 16 --repetitions 100000
 ```
 
 !!! note
-    By default, self\_test will use the network interface selected by the agent.
-    This can be forced by setting the OFI\_INTERFACE and OFI\_DOMAIN environment
-    variables manually. e.g. export OFI\_INTERFACE=eth0; export OFI\_DOMAIN=eth0
-    or export OFI\_INTERFACE=ib0; export OFI\_DOMAIN=mlx5_0
+    By default, `self_test` will use the network interface selected by the agent.
+    This can be forced by setting the `OFI_INTERFACE` and `OFI_DOMAIN` environment
+    variables manually. e.g. export `OFI_INTERFACE=eth0; export OFI_DOMAIN=eth0`
+    or `export OFI_INTERFACE=ib0; export OFI_DOMAIN=mlx5_0`
 
 !!! note
     Depending on the HW configuration, the agent might assign a different
-    network interface to the self_test application depending on the NUMA node
+    network interface to the `self_test` application depending on the NUMA node
     where the process is scheduled. It is thus recommended to use taskset to bind
-    the self_test process to a specific core. e.g. taskset -c 1 self_test ...
+    the `self_test` process to a specific core. e.g. taskset -c 1 `self_test` ...
 
 ### Example: Cross-Servers
 
-To run self\_test in cross-servers mode:
+To run `self_test` in cross-servers mode:
 
 ```bash
 
-$ self_test -u --group-name daos_server --endpoint 0-<MAX_SERVER-1>:0 \
-  --master-endpoint 0-<MAX_RANK>:0-<MAX_TAG> \
+$ self_test --use-daos-agent-env --group-name daos_server \
+  --endpoint 0-<MAX_SERVER-1>:0 --master-endpoint 0-<MAX_RANK>:0-<MAX_TAG> \
```

Comment:
you already use MAX_RANK when describing master-endpoint.  should "--endpoint" be also described using MAX_RANK instead of MAX_SERVER-1 ?

Reply:
- [ ] Replace `MAX_SERVER` with `MAX_RANK`